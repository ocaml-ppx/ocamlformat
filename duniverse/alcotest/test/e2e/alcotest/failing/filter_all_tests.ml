(** Ensure that filters which eliminate all tests are rejected. *)

let () =
  Alcotest.run __FILE__
    [
      ("foo", [ Alcotest.test_case "1" `Quick (fun () -> assert false) ]);
      ("bar", [ Alcotest.test_case "2" `Quick (fun () -> assert false) ]);
    ]
