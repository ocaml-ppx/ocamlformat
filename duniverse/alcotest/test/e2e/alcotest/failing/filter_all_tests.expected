Invalid request (no tests to run, filter skipped everything)!
Testing `test/e2e/alcotest/failing/filter_all_tests.ml'.
This run has ID `<uuid>'.

